atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/UniqueSchemaMutationLockTable.java

```diff
@@ -84,11 +85,28 @@ private TableReference getSingleTable() throws TException {
         Set<TableReference> lockTables = schemaMutationLockTables.getAllLockTables();
 
         if (lockTables.size() > 1) {
-            throw new IllegalStateException("Multiple schema mutation lock tables have been created.\n"
-                    + "This happens when multiple nodes have themselves as lockLeader in the configuration.\n"
-                    + "Please ensure the lockLeader is the same for each node, stop all Atlas clients using this "
-                    + "keyspace, restart your cassandra cluster and run the clean-cass-locks-state cli command.\n"
-                    + "The tables that clashed were: " + lockTables);
+            // Try to recover by checking if other lock tables are empty and dropping them.
+
+            Set<TableReference> emptyLockTables = lockTables.stream()
+                    .filter(schemaMutationLockTables::isLockTableEmpty)
+                    .collect(Collectors.toSet());
+
+            if (lockTables.size() - emptyLockTables.size() > 1) {
+                throw new IllegalStateException("Multiple schema mutation lock tables have been created.\n"
+                        + "This happens when multiple nodes have themselves as lockLeader in the configuration.\n"
+                        + "Please ensure the lockLeader is the same for each node, stop all Atlas clients using this "
+                        + "keyspace, restart your cassandra cluster and run the clean-cass-locks-state cli command.\n"
+                        + "The tables that clashed were: " + lockTables);
+            }
+
+            if (lockTables.size() == emptyLockTables.size()) {
+                emptyLockTables.remove(emptyLockTables.iterator().next());
```

Comment:
What happens if multiple nodes are running this check at once? We should ensure that the "lucky" table is chosen consistently (e.g. lexicographic ordering).